src/content/docs/infrastructure/prometheus-integrations/install-configure-openmetrics/install-Flink-prometheus-openMetrics.mdx

```diff
@@ -0,0 +1,73 @@
+metaDescription: Get dashboards of all your most important Flink metrics with our quickstart. 
+---
+
+import instantobservabilityFlink from 'images/instantobservabilityFlink.png'
+
+With our Apache Flink dashboards, you can easily track your Flink logs, keep an eye on your instrumentation sources, and get an overview of uptime and downtime for all your applications' instances. Built with our infrastructure agent and our Prometheus OpenMetrics integration, Flink dashboards take advantage of OpenMetrics endpoint scraping, so you can view all your most important data, all in one place.
+
+<img
+    src={instantobservabilityFlink}
+    title="Flink dashboard landing page"
+    alt="A screenshot of a dashboard with Flink metrics."
+/>
+<figcaption>
+After setting up Flink with New Relic, your data will display in dashboards like these, right out of the box. 
+</figcaption> 
+
+## Install and Configure Prometheus OpenMetrics
+
+Before getting Flink data into New Relic, first install our infrastructure agent, then expose your metrics by installing Prometheus OpenMetrics. 
+
+* Follow our [guided install](https://one.newrelic.com/nr1-core?account=3501165&state=a33ce5fc-d32b-e0b8-3941-30f13e4bc00b) to instrument your system with the infrastructure agent. Or if you prefer, you can [install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux) via the command line.
+* Install our [Prometheus OpenMetrics integration](/docs/infrastructure/prometheus-integrations/install-configure-openmetrics/install-update-or-uninstall-your-prometheus-openmetrics-integration).
+
+After you've installed Prometheus OpenMetrics, then you need to configure the `nri-prometheus-config.yml` file. Add the following snippet anywhere to the file:
+
+```yml
+integrations:
+- name: nri-prometheus
+    config:
+    standalone: false
+    # Defaults to true. When standalone is set to `false`, `nri-prometheus` requires an infrastructure agent to send data.
+    emitters: infra-sdk
+    # When running with infrastructure agent emitters will have to include infra-sdk
+    cluster_name: <var>YOUR_CLUSTER_NAME_HERE</var>
+    # Match the name of your cluster with the name seen in New Relic. 
+    targets:
+        - description: <var>YOUR_DESCRIPTION_HERE</var>
+            urls: ["https://192.168.3.1:2379", "https://192.168.3.2:2379", "https://192.168.3.3:2379"]
+            tls_config:
+                ca_file_path: "/etc/etcd/etcd-client-ca.crt"
+                cert_file_path: "/etc/etcd/etcd-client.crt"
+                key_file_path: "/etc/etcd/etcd-client.key"
+    verbose: false
+    # Defaults to false. This determines whether or not the integration should run in verbose mode.
+    audit: false
+    # Defaults to false and does not include verbose mode. Audit mode logs the uncompressed data sent to New Relic and can lead to a high log volume.
+    scrape_timeout: <var>YOUR_TIMEOUT_DURATION</var>
```

Comment:
Scrape timeout can also be commented out as it is not mandatory to integrate Apache Flink. Here, "30s" is the default value
```
      # scrape_timeout: "30s"
```